src/main/java/seedu/address/model/person/MedCon.java

```diff
@@ -12,18 +12,18 @@
 public class MedCon implements Comparable<MedCon> {
 
     public static final String VALIDATION_REGEX = "\\p{Alnum}+(\\s\\p{Alnum}+)*";
-    public final String value;
+    public final String medConName;
```

Comment:
Good change of name to better fix the intended purpose!

Reply:
yep